GCSViews/CameraView.cs

```diff
@@ -86,12 +107,12 @@ public CameraView()
             instance = this;
 
             // Video control
-            VideoControl = CameraHandler.Instance.CameraVideoControl;
+            //VideoControl = CameraHandler.Instance.CameraVideoControl;
             VideoControlDLLVersion = CameraHandler.Instance.StreamDLLVersion;
             CameraStreamIP = SettingManager.Get(Setting.CameraStreamIP);
             CameraStreamPort = int.Parse(SettingManager.Get(Setting.CameraStreamPort));
             FetchHudDataTimer.Interval = 100; // 10Hz
-            FetchHudDataTimer.Tick += (sender, eventArgs) => FetchHudData();
+            //FetchHudDataTimer.Tick += (sender, eventArgs) => FetchHudData();
```

Comment:
A FetchHudData() azért került ki egy timer-be, mert ha minden új frame betöltésekor lefutott, drasztikusan lelassította a képet